fe/fe-core/src/main/java/org/apache/doris/common/proc/ProcService.java

```diff
@@ -38,7 +38,7 @@ private ProcService() {
         root.register("backends", new BackendsProcDir(Catalog.getCurrentSystemInfo()));
         root.register("dbs", new DbsProcDir(Catalog.getCurrentCatalog()));
         root.register("jobs", new JobsDbProcDir(Catalog.getCurrentCatalog()));
-        root.register("statistic", new StatisticProcDir(Catalog.getCurrentCatalog()));
```

Comment:
better not to modify the default value. Some online system rely on it. We should make it forward-compatible, in most cases.

Reply:
That makes sense, I'll change it back right now.